packages/web/src/components/stripe-on-ramp-modal/StripeOnRampModal.module.css

```diff
@@ -1,6 +1,6 @@
 .stripeModalWrapper {
   z-index: 99999999999999999999;
-  position: absolute;
+  position: fixed;
```

Comment:
Looking below, seems like we already use a lot styling to center within the viewport. So this is probably safe.